Sources/AardvarkLoggingUI/Log Viewing/ARKLogTableViewController.m

```diff
@@ -27,6 +27,12 @@
 #import "ARKScreenshotViewController.h"
 #import "UIActivityViewController+ARKAdditions.h"
 
+typedef NS_ENUM(NSUInteger, ARKLogExportOption) {
```

Comment:
can you help me understand why this change was needed? This code looks reasonable but I'm not following why this was required.

Reply:
This is one of @JustinDSN's commits, so he could explain the thinking behind the change better than I can. The short version is that `UIActionSheet` and `UIPopoverController` are deprecated in iOS 8.3 and iOS 9, respectively, so this replaces their usage with `UIAlertController`s.

This specific change (the enum of export options) could be implemented as two separate methods fairly easily. This route adds a bit more structure in order to avoid repeating a bit of logic. I don't have strong feelings either way given it's not a ton of structure but also not a ton of repeated logic.

Reply:
You summarized the specific trade offs I was trying to balance here. It's really good feedback to see the feedback on repeated logic and that would have been acceptable. Thanks for representing the thought process as well as seeing both sides of the same coin.

Reply:
A comment explaining the why would be helpful for future maintainers 🙂

Reply:
Done here: https://github.com/square/Aardvark/pull/94